### ISSUE TITLE
Nested query not parsed properly in v702

### ISSUE THREAD

**Issue by fdupret** [abaplint/abaplint#3999]
Regression from https://github.com/abaplint/abaplint/pull/3983? @cohomology
Worked fine in 2.119.27.

```abap
SELECT foo
  FROM bar
  INTO TABLE table
  WHERE foo  IN ( SELECT foo
                    FROM bar ).
```
abaplint returns `Statement does not exist in ABAPv702(or a parser error), "SELECT"(parser_error)`
Bumping abaplint rule to v740 and abaplint doesn't complain anymore. I can confirm the syntax works fine on a v702 ABAP system.

**Comment by larshp** [comment]
@fdupret try again, new version published

**Comment by fdupret** [comment]
Thanks for the quick fix!
